store/storetest/user_store.go

```diff
@@ -296,6 +297,43 @@ func testUserStoreGet(t *testing.T, ss store.Store) {
 	})
 }
 
+func testUserStoreGetMany(t *testing.T, ss store.Store) {
+	users := make([]*model.User, 2)
+	for i := 0; i < 2; i++ {
+		u := &model.User{
+			Email: MakeEmail(),
+		}
+		user, err := ss.User().Save(u)
+		require.Nil(t, err)
+		users[i] = user
+	}
+	defer func() {
+		require.Nil(t, ss.User().PermanentDelete(users[0].Id))
+		require.Nil(t, ss.User().PermanentDelete(users[1].Id))
+	}()
+
+	t.Run("fetch empty ids", func(t *testing.T) {
+		res, err := ss.User().GetMany([]string{})
+		require.Nil(t, err)
```

Comment:
It is more correct to use `require.(No)Error` family of functions to check for variables of type `error`.

Reply:
I've tried to keep it consistent with the rest of the tests in this file. Will amend all of those to use `NoError`

Reply:
Yes, this is an artifact from the apperror migration from model.AppError to error. We actually need to change all `require.Nil` of error to `require.NoError`.

Reply:
Already done for this file :)